tests/llm/fixtures/test_ask_holmes/120_new_relic_traces2/deployment.yaml

```
@@ -0,0 +1,411 @@
+  requirements.txt: |
+    opentelemetry-sdk>=1.16,<2.0
+    opentelemetry-exporter-otlp>=1.16,<2.0
+    opentelemetry-instrumentation-flask>=0.36b0,<0.42
+    opentelemetry-instrumentation-requests>=0.36b0,<0.42
+    opentelemetry-instrumentation-logging>=0.36b0,<0.42
+    opentelemetry-semantic-conventions>=0.36b0,<0.42
+    requests>=2.24,<3.0
+    Flask>=2.2,<3.0
+
+
+  requirements.txt: |
```

YAMLlint (1.37.1)

[error] duplication of key "requirements.txt" in mapping

(key-duplicates)